src/main/java/br/com/caelum/vraptor/util/jpa/extra/ParameterLoader.java

```diff
@@ -69,32 +73,29 @@ public ParameterLoader(EntityManager em, HttpServletRequest request, ParameterNa
 		this.flash = flash;
 	}
 
+	@Accepts
 	public boolean containsLoadAnnotation(ControllerMethod method) {
```

Comment:
I mean just change your method signature to:

```java
 @Accepts
 public boolean containsLoadAnnotation() { ... }
````

and ask `ControllerMethod` via constructor